VocaDbWeb/Views/Event/Details.cshtml

```diff
@@ -232,7 +232,8 @@
 		$("#manageTags").button({ icons: { primary: 'ui-icon-wrench' } });
 
 		var loggedUserId = @UserContext.LoggedUserId;
-		var urlMapper = new vdb.UrlMapper('@UrlMapper.HostAddress');
+		var rootPath = '/';

```

Comment:
Hmm why did you hardcore rootpath to '/' instead of using the config parameter?

Reply:
Fixed, but @UrlMapper.HostAddress should not be used because it generates a non-SSL URL (e.g. http://localhost:39390/ instead of https://localhost:44399/).